docs/stdlib/datetime.rst

```diff
@@ -550,7 +548,7 @@ EdgeDB stores and outputs timezone-aware values in UTC.
                               -> cal::local_datetime
                           cal::local_date + duration -> cal::local_datetime
 
-    Time interval addition.
+    Adds two variations of any date/time type into one.
```

Comment:
```suggestion
    Adds any two datetime or duration values.
```

We're not adding the types here, but if I just remove the word "type" from the rewrite, it sounds as though you need two variations of the _same_ value to use the operator.